src/Libraries/Nop.Services/Helpers/DateTimeHelper.cs

```diff
@@ -156,16 +156,16 @@ public virtual DateTime ConvertToUtcTime(DateTime dt, TimeZoneInfo sourceTimeZon
         /// A task that represents the asynchronous operation
         /// The task result contains the customer time zone; if customer is null, then default store time zone
         /// </returns>
-        public virtual async Task<TimeZoneInfo> GetCustomerTimeZoneAsync(Customer customer)
+        public virtual TimeZoneInfo GetCustomerTimeZone(Customer customer)

```

Comment:
No need to change this method to the sync version. Let's leave async for consistency.